packages/terra-clinical-result/CHANGELOG.md

```diff
@@ -3,7 +3,9 @@ ChangeLog
 
 Unreleased
 -----------------
+### Fixed
+* Fix bug when ClinicalBloodPresure was provided both a systolic and diastolic no data, presented error
```

Comment:
The sentence seems incomplete.

Reply:
2nd round attempt https://github.com/cerner/terra-clinical/pull/614/commits/b0b5305327e8c17a49550348602d6ea79f39c368#diff-534a7c4c0b2429787b66bb7aac9cade0R7